encoders/numeric/TruncatedSVDEncoder/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM jinaai/jina:1.1.2
```

Comment:
Please, remove the version tag of jina. This is updated by dependabot when building new versions